packages/storybook/stories/ChatComposite/snippets/CustomizeBehavior.snippet.tsx

```diff
@@ -38,5 +38,7 @@ export const ContosoChatContainer = (props: ContainerProps): JSX.Element => {
     createAdapter();
   }, [props]);
 
-  return <>{adapter ? <ChatComposite adapter={adapter} /> : <h3>Loading...</h3>}</>;
+  return (
+    <>{adapter ? <ChatComposite adapter={adapter} options={{ showParticipantPane: true }} /> : <h3>Loading...</h3>}</>
```

Comment:
Minor: Could this just be?
```suggestion
    adapter ? <ChatComposite adapter={adapter} options={{ showParticipantPane: true }} /> : <h3>Loading...</h3>
```
similar for the other changes

Reply:
True. I'm lazy to make that change here. Will upload another PR, hopefully remembering :)